test/WebSites/RazorWebSite/Views/ViewEngine/ViewWithRelativePath.cshtml

```diff
@@ -12,4 +12,4 @@
 }
 ViewWithRelativePath-content
 <partial>@await Html.PartialAsync("../Shared/_PartialThatSetsTitle.cshtml")</partial>
-@await Component.InvokeAsync("ComponentWithRelativePath", person)
\ No newline at end of file
+@await Component.InvokeAsync("ComponentWithRelativePath", new { person })
```

Comment:
Not for this PR, but we could try to be clever in this case and attempt to make it work if the user does

`@await Component.InvokeAsync("ComponentWithRelativePath", person)` (similarly with a params array). The user experience for single parameters is a lot worse than what it used to be before.